std_msgs/Header header 
NodeState[] nodes
